app/controllers/boards_controller.rb

```diff
@@ -22,13 +22,25 @@ def edit
 
   def update
     @board = Board.find(params[:id])
-    if @board.update
+    if @board.update({title: params[:title]})
       redirect_to boards_path
     else
       render :edit
     end
   end
 
+  def destroy
+    Board.find(params[:id]).destroy
+    flash[:success] = "Board deleted"
+    redirect_to boards_path
```

Comment:
mozna uzyc flasha w ten sposób:
redirect_to boards_path, notice: 'Board deleted' 
i to jest takie standardowe railsowe powiadomienie.